src/docs/tutorial-transaction-process/customize-pricing/index.md

```diff
@@ -0,0 +1,482 @@
+in this example as the input, the `cleaningFee` will be Money object when we get
+that from `values`. Money object can't be used directly as public data so we
+need to create a JSON object with keys `amount` and `currency` and use the JSON
+object in the API call.
+
+> **Note:** `price` attribute is one of the listing's default attributes so it's
+> passed to Marketplace API directly unlike the new public data `cleaningFee`
+> which needs to be under the `publicData` key.
+
+```jsx
+      onSubmit={values => {
+        const { price, cleaningFee = null } = values;
+
+        const updatedValues = {
+          price,
+          publicData: {
+            cleaningFee: { amount: cleaningFee.amount, currency: cleaningFee.currency },
+          },
+        };
+        onSubmit(updatedValues);
+      }}
+```
+
+### Initialize the form
+
+Next we want to pass inital values for `price` and `cleaningFee`. For this we
+need to get the `cleaningFee` from listing attributes under the `publicData`
+key. Also, because `FieldCurrencyInput` expects the value to be Money object we
+need to convert the value we get from Marketplace API back to instance of Money.
+
+```jsx
+const { price, publicData } = currentListing.attributes;
+const cleaningFee =
+  publicData && publicData.cleaningFee ? publicData.cleaningFee : null;
+const cleaningFeeAsMoney = new Money(cleaningFee.amount, cleaningFee.currency);
+const initialValues = { price, cleaningFeeAsMoney };
+```
+
+### Add input component
+
+We want to be able to save the amount of listing's cleaning fee so we add new
+`FieldCurrencyInput` to the _EditListingPricingForm_. The id and name of this
+field will be `cleaningFee`. Adding this fee will be optional so we don't want
+to add any `validate` param to the `FieldCurrencyInput` like there is in the
+`price` input.
+
+```shell
+└── src
+    └── forms
+        └── EditListingPricingForm
+            └── EditListingPricingForm.js
+```
+
+```jsx
+...
+
+<FieldCurrencyInput
+    id="price"
+    name="price"
+    className={css.priceInput}
+    autoFocus
+    label={pricePerUnitMessage}
+    placeholder={pricePlaceholderMessage}
+    currencyConfig={config.currencyConfig}
+    validate={priceValidators}
+  />
+
+<FieldCurrencyInput
+  id="cleaningFee"
+  name="cleaningFee"
+  className={css.cleaningFeeInput}
+  autoFocus
+  label={cleaningFeeMessage}
+  placeholder={cleaningFeePlaceholderMessage}
+  currencyConfig={config.currencyConfig}
+/>
+...
+```
+
+After adding the new translations keys and some padding to price input in CSS
+file, the EditListingPricingPanel should look something like this:
+
+![EditListingPricePanel](./editlistingpricepanel.png)
+
+## Update BookingDatesForm
+
+In our example the cleaning fee is optional and users can select that as an
+add-on to their booking. In this section, we will add the UI component for
+selecting the cleaning fee and pass the information about the user's choice to
+the FTW-backend.
```

Comment:
"FTW-backend" sounds confusing to me. We have FTW-daily and FTW-hourly - is the FTW-backend a new repository?

- the backend of our client app.
- the backend of FTW template.
- the backend of FTW-daily.

Reply:
Change the other 6 instances too.

Reply:
👍  I changed that to "the backend of our client app" instead